docs/backend/upgrading/version-specific-migration/upgrade-to-61.md

```diff
@@ -117,3 +117,44 @@ If you have an existing Plone 5.2 or 6.0 site and you migrate to 6.1, then migra
 -   If the `plone.app.discussion` Python package is _not_ in your setup, but the site has existing comments (discussions), then the migration code stops with an error.
     Apparently you _were_ using comments in your site.
     Add the `plone.app.discussion` package to your dependencies, and run the migration again.
+
+
+## Distributions
+
+Plone 6.1 introduces the concept of a Plone {term}`distribution`.
+A Plone distribution is a Python package that defines specific features, themes, add-ons, and configurations that get activated when creating a Plone site.
+Now it is available in core Plone as the recommended way for {doc}`creating a new Plone site </install/add-site>`.
+
+```{seealso}
+For more information, see {doc}`/conceptual-guides/distributions`.
+```
+
+Distributions are optional.
+If your project only uses the `Products.CMFPlone` Python package, you can still create a Plone site in the old way.
+Consider, however, the following differences from distributions and from Plone 6.0.
+
+-   The configuration form is simpler and shorter.
+-   The created site has no content, and therefore no {guilabel}`News` or {guilabel}`Events` folders.
+-   You must activate add-ons through the {guilabel}`Add-ons` control panel.
+
+There are a few things you should consider when upgrading a project to, or making an add-on compatible with, Plone 6.1:
+
+-   In general, you don't need to change anything.
+    Your existing site will keep working.
+    But adding a new site may change in the ways described earlier.
+-   Do you want to use the `Products.CMFPlone` package (0 distributions), either `plone.volto` or `plone.classicui` (1 distribution), or `Plone` (2 distributions)?
+-   If your site uses the default Volto frontend, you will already have `plone.volto` as a dependency.
+    This can stay the same.
+-   If your site uses only the `Products.CMFPlone` package, the frontend is Classic UI.
```

Comment:
I thought all Plone sites use `Products.CMFPlone`? Should this be:

```suggestion
-   If your site uses Classic UI for the frontend, you will already have `Products.CMFPlone` as a dependency and not `plone.volto`.
```

Reply:
This line is talking about the situation where your project _only_ installs `Products.CMFPlone` and not `Plone`. Your suggestion as worded is not quite the point Maurits was trying to make here. Instead I suggest:
```suggestion
-   If your site depends on the `Products.CMFPlone` package without the `Plone` or `plone.volto` packages, the frontend is Classic UI.
```